packages/wallets/src/components/WalletsErrorScreen/__tests__/WalletsErrorScreen.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import WalletsErrorScreen from '../WalletsErrorScreen';
+
+describe('WalletsErrorScreen', () => {
+    afterAll(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should show the correct title and icon with default message', () => {
+        render(<WalletsErrorScreen />);
+        expect(screen.getByTestId('dt_error_icon')).toBeInTheDocument();
+        expect(screen.getByText('Oops, something went wrong!')).toBeInTheDocument();
+        expect(
+            screen.getByText('Sorry an error occurred. Please try accessing our cashier page again.')
+        ).toBeInTheDocument();
+    });
+
+    it('should show the message passed as prop', () => {
+        render(<WalletsErrorScreen message='Error message from props' />);
+        expect(screen.getByText('Error message from props')).toBeInTheDocument();
+    });
+
+    it('should reload the page when Try again button is clicked', () => {
+        const reloadMock = jest.fn();
+        Object.defineProperty(window, 'location', {
+            value: { reload: reloadMock },
+            writable: true,
+        });
+        render(<WalletsErrorScreen />);
+        screen.getByText('Try again').click();
```

Comment:
hey @farhan-nurzi-deriv 
one small suggestion
could we use getByRole for button element ?
According RTL guideline they prioritise getByRole for elements such button e.g.. Pls have a look [here](https://testing-library.com/docs/queries/about#:~:text=Yes-,Priority%E2%80%8B,button%27%2C%20%7Bname%3A%20/submit/i%7D).%20Check%20the%20list%20of%20roles.,-getByLabelText%3A%20This%20method)

Reply:
Sure @heorhi-deriv will change it 👍